DiscordIan/Properties/serviceDependencies.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What is this file for? Is it necessary?

Reply:
Probably not, when I pulled your branch it said I needed to download a secrets update and it created those.

Reply:
Weirdness. I'll investigate this further.